src/components/filterContainer/FilterContainer.tsx

```diff
@@ -7,11 +7,15 @@ import { FilterType } from './FilterLabel.types'
 
 export const calculateIsSelected = (
   filterType: FilterType,
-  filterName: string,
+  filterName: string | string[],
```

Comment:
If this is changing, we will also need this `filterName` to be accepted by `radio`, else the component doesn't fully work.

Reply:
We'll also require a new story (per `variant`) to highlight the usage if this is changing.

Reply:
I added a story in `CategoryFilter.stories` with `ENTRIES_WITH_CATEGORY_ARRAY` as the entries, the difference being that the categories in the entries now have an array, it seemed before it only handled single category entries.